plan/stats.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+import (
+	"github.com/ngaut/log"
+	"github.com/pingcap/tidb/expression"
+)
+
+type statsProfile struct {
```

Comment:
Add comment for the struct. For example why we need this and when will it be used.